yorkie/backend/backend.go

```diff
@@ -43,6 +45,40 @@ type Config struct {
 
 	// AuthorizationWebhookURL is the url of the authorization webhook.
 	AuthorizationWebhookURL string
+
+	authorizationWebhookMethods map[types.Method]bool
```

Comment:
I missed it in the previous work, but it would be nice to be able to configure it with a config file. How about leaving raw settings in the config?

```suggestion
	AuthorizationWebhookURL string `json:"AuthorizationWebhookURL"`

	AuthorizationWebhookMethods []string{} `json:"AuthorizationWebhookMethods"`
```